lib/Doctrine/Common/Annotations/AnnotationRegistry.php

```diff
@@ -32,34 +29,39 @@
      *
      * This autoloading mechanism does not utilize the PHP autoloading but implements autoloading on its own.
      *
-     * @var array
+     * @var string[][]|string[]|null[]
      */
-    static private $autoloadNamespaces = array();
+    static private $autoloadNamespaces = [];
 
     /**
      * A map of autoloader callables.
      *
-     * @var array
+     * @var callable[]
      */
-    static private $loaders = array();
+    static private $loaders = [];
 
     /**
-     * @return void
+     * An array of classes which cannot be found
+     *
+     * @var null[] indexed by class name
```

Comment:
Yeah, that's intentional, as a boolean can misrepresent two states, while here we just want to represent whether a key exists or not